roles/serviceassurance/defaults/main.yml

```diff
@@ -2,11 +2,15 @@
 # set default enablement
 metrics_enabled: true
 events_enabled: false
+highAvailabilityEnabled: false
```

Comment:
So with Ansible Operators, a camelCase will be automatically converted into snake_case. That's why everything is defined here in the defaults as the snake_case in order to just have a backup conversion in case the camelCase is defined incorrectly in the CR.

Change this to `high_availability_enabled: false`